### PR TITLE
[WIP] libxml2: update to 2.9.10.

### PR DESCRIPTION
**Pull request by travankor**

### CODE REVIEW

File: srcpkgs/libxml2-python/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Template file for 'libxml2-python'
 pkgname=libxml2-python
-version=2.9.9
-revision=3
+version=2.9.10
+revision=1
 wrksrc="${pkgname%-python}-${version}"
 build_wrksrc=python
-build_style=python-module
+build_style=python3-module
```
Comment (Hoshpak) [inline srcpkgs/libxml2-python/template]: this doesn't seem right. It will leave the `-python` package empty except for the license file.

Comment (xtraeme) [reply srcpkgs/libxml2-python/template]: what @Hoshpak said. maybe add a subpkg that depends on python3 (meta)

File: srcpkgs/libxml2-python/template [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
 }
 
 libxml2-python3_package() {
-	short_desc+="${short_dest/Python2/Python3}"
-	pkg_install() {
-		vmove "usr/lib/python3*"
-		vlicense ../COPYING
-	}
+	depends="libxml2-python"
+	short_desc+=" - transitional dummy package"
```
Comment (Hoshpak) [inline srcpkgs/libxml2-python/template]: This is also wrong. It breaks the naming convention and replaces one package with another one that doesn't offer the same functionality and thus will break packages depending on it.

I think the best course of actions would be to make libxml2-python3 a separate package with its own template which will have to happen anyway when python2 is dropped. Then issues with the python 2 bindings can be handled separately.

Comment (travankor) [reply srcpkgs/libxml2-python/template]: which ones depend on python2? I only found `gnome-doc-utils` and bumped it to python3 with a patch.

Comment (travankor) [reply srcpkgs/libxml2-python/template]: `bamf`, `libGL`, and `libxslt` seem to be the other three

Comment (Hoshpak) [reply srcpkgs/libxml2-python/template]: Indeed. Removing the python2 version is alright if nothing else depends on it, but not if it is done this way. You should just remove libxml2-python and make libxml2-python3 a separate package.

**Pull request closed without merging**